samples/vue/sitecore/definitions/components/ContentBlock.sitecore.js

```diff
@@ -6,7 +6,7 @@ import { CommonFieldTypes, SitecoreIcon, Manifest } from '@sitecore-jss/sitecore
  * This function is invoked by convention (*.sitecore.js) when `jss manifest` is run.
  * @param {Manifest} manifest Manifest instance to add components to
  */
-export default function(manifest) {
+export default function (manifest) {
```

Comment:
This seems like an unwanted linting configuration, do we want this space here?
I don't think we have this setting in our other samples.

Reply:
It's fine, everywhere we have space between function and (. It happened because prettier was upgraded

Reply:
In react sample - the same

Reply:
Ah I see, it does this for default exports and not named. Maybe we should standardize that at some point, but works for me for now.